### ISSUE TITLE
The setting of the parameter - -in-gn when using the metatranscriptomic reads

### ISSUE THREAD

**Issue by B-1991-ing** [AnantharamanLab/METABOLIC#89]
Hi Zhichao,

What should we take as the input for the parameter - -in-gn when we set **-rt metaT**? Or we don't need to use the parameter - -in-gn when taking the metatranscriptomic reads?

Best,

Bing

**Comment by ChaoLab** [comment]
The input of "--in-gn" should be the location of FOLDER containing the fasta files of input genomes

**Comment by B-1991-ing** [comment]
But, the MAG are not assembled from the metatranscriptomic reads, they are relatively independent with the metatranscriptomic data.

**Comment by B-1991-ing** [comment]
What we want to see is how both gene abundance – and especially gene expression – of the different genes.

**Comment by ChaoLab** [comment]
> 

Metatranscriptomic reads can be mapped onto MAGs since the genomic DNA will be transcribed into RNA. So biologically, this makes sense

**Comment by B-1991-ing** [comment]
Thank you for your brief explanation, I will give it a try.

**Comment by ChaoLab** [comment]
> Hi Zhichao,
> 
> What is the difference between taking **the assembled MAGs & the corresponding metagenome reads where the MAG were assembled from** and taking **the assembled MAGs & the metatranscriptome reads** as the input of the metabolic software? I did both.
> 
> Does the MW-score_result.txt from **the assembled MAGs & the corresponding metagenome reads** can let us look all genes’ expression in each pathway at the genome level? Does the MW-score_result.txt from **the assembled MAGs & the corresponding metatranscriptome reads** can let us look all genes’ expression in each pathway at the transcriptome level?
> 
> Or we maybe need to do the gene expression by ourself using the **All_gene_collections_mapped.depth.txt** and **All_gene_collections_transcript_coverage.txt**, respectively?
> 
> Best,
> 
> Bing

Yes, you will need to do gene expression by yourself using those two txt files.

MW-score gave you another metric to compare between different samples.

**Comment by B-1991-ing** [comment]
> Yes, you will need to do gene expression by yourself using those two txt files.

What kind of figure is best to display the gene expression in one pathway at a ‘genome’ and ‘transcriptome’ level? Hetamap or Manhattan?

**Comment by ChaoLab** [comment]
I have no experience in using Manhattan chart (maybe it is new), I suggest heatmap

**Comment by B-1991-ing** [comment]
I plotted the gene coverage of each MAG in each 'genome' or 'metagenome' in two heatmaps. 

The All_gene_collections_transcript_coverage.txt is the gene coverage of all MAG contigs across all metatranscriptome samples? The we can see the gene coverage of all genes across all metatranscriptome samples also?

**Comment by ChaoLab** [comment]
It is the gene (transcript for metatranscriptome) coverage of all MAG genes. METABOLIC only does metagenomic/metatranscriptomic mapping based on all genes
